### PR TITLE
Fixed close issue for error msg of login

### PR DESCRIPTION
**Pull request by ghost**
This is the continue fixing  of SCRD-4453

### CODE REVIEW

**Review by jeremy-moffitt — DISMISSED** [review]
this works, and seems simple enough, its a little unfortunate that the z-index is set in 2 different related css classes

**Comment by guglo** [comment]
this fixes it but it is not the best fix, will figure out the a better way.

**Comment by guglo** [comment]
@jeremy-moffitt  thanks for the update, tested it, it works well.

File: src/styles/components/messages.less [hunk 1]
```diff
@@ -14,17 +14,15 @@
...
-}
 
 .notification-message-container {
   position: absolute;
   top: 1em;
   right: 1em;
-  &.login {
+  &.login,
+  .notification-message{
     z-index: 1111;
```
Comment (GarySmith) [inline src/styles/components/messages.less]: Is there some rationale behind this very large z-index? I don't know much about z-indexes, but this looks like some special value.

Comment (jeremy-moffitt) [reply src/styles/components/messages.less]: its what was there before... not a great answer I know

Comment (jeremy-moffitt) [reply src/styles/components/messages.less]: I tried again with a z-index of 20, and it works for the login page... going to check the other places its used

**Pull request merged**